assets/js/base/components/cart-checkout/totals/footer-item/index.js

```diff
@@ -37,34 +38,46 @@ const TotalsFooterItem = ( { currency, values } ) => {
 
 	const parsedTaxValue = parseInt( totalTax, 10 );
 
+	const TaxLabel = ( SHOW_TAXES && parsedTaxValue !== 0 && (
+		<p className="wc-block-components-totals-footer-item-tax">
+			{ createInterpolateElement(
+				__(
+					'Including <TaxAmount/> in taxes',
+					'woo-gutenberg-products-block'
+				),
+				{
+					TaxAmount: (
+						<FormattedMonetaryAmount
+							className="wc-block-components-totals-footer-item-tax-value"
+							currency={ currency }
+							value={ parsedTaxValue }
+						/>
+					),
+				}
+			) }
+		</p>
+	) ) || <></>;
+
+	const filteredTotalDescription = createInterpolateElement(
+		__experimentalApplyCheckoutFilter( {
+			filterName: 'totalDescription',
+			defaultValue: __( '<TaxLabel/>', 'woo-gutenberg-products-block' ),
+			extensions: cart.extensions,
+			arg: { cart },
+			validation: ( value ) => mustContain( value, '<TaxLabel/>' ),
+		} ),
+		{
+			TaxLabel,
+		}
+	);
```

Comment:
This is an interesting way of doing this. The only concern I have here is that what happens if in future designs we want to have another component represented in this area that "must" be contained in the returned string from the filter? There's a tight coupling to the current requirements of the `totalDescription` having a `<TaxLabel />` string in it for interpolation.

What about having a completely separate filtered string (default to empty string) that is appended after the description and then passed along? The `description` prop would then be composed by the filtered string and `<TaxLabel />` component:

```jsx
const totalsDescription = (
	<>
		<TaxLabel />
		{ filteredDescription }
	</>
);
```

Reply:
Of course, this might suggest a slot/fill is the better option here too. I'm on the fence.

Reply:
As mentioned in the [comment above](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5153#:~:text=ralucaStan%20reviewed-,13%20minutes%20ago,-View%20changes), I think it's better to separate the taxes all together, as they will be forever present and will less likely change with time.

I was also thinking about wether a Slot / Fill would be a better implementation, considering thatwith a filter, the newly added 'description' zone could just contain a string, and a Slot would offer more flexibility (HTML content or so).

Reply:
If we only want to allow modification to the "main" `TotalsItem` label after the description and be strict about the position, then maybe Slot/Fill will be better. The filter in its present state allows for the extension to choose whether to output before or after the label.

Reply:
Extensions that offer the ability to pay in instalments will benefit from this being a slot fill for the HTML content a potential linking to conditions, terms or other content.

Also, we would like to have control over CSS separately, as this extensions would be wanting to highlight an important content in it separately from the tax.